File: fix_types.py
import re

with open('src/components/ExhibitionModal.tsx', 'r') as f:
    text = f.read()

# Remove duplicate "sculptures"
text = re.sub(r'(\s*"sculptures": "Sculpture",\n)+', r'\n    "sculptures": "Sculpture",\n', text)
text = re.sub(r'(\s*"photography": "Photography",\n)+', r'\n    "photography": "Photography",\n', text)
text = re.sub(r'(\s*"posters": "Posters",\n)+', r'\n    "posters": "Posters",\n', text)

with open('src/components/ExhibitionModal.tsx', 'w') as f:
    f.write(text)

with open('src/components/InteractiveGlobeMap/InteractiveGlobeRealModal.tsx', 'r') as f:
    text2 = f.read()

text2 = re.sub(r"(\s*'egyptian-museum-cairo-collection': '/data/egyptian-museum-cairo-collection\.json',\n)+", r"\n          'egyptian-museum-cairo-collection': '/data/egyptian-museum-cairo-collection.json',\n", text2)
text2 = re.sub(r"(\s*'conde-paintings': '/data/musee-conde-collection\.json',\n)+", r"\n          'conde-paintings': '/data/musee-conde-collection.json',\n", text2)

wimport re

with open('src/componeiv
with opp/InteractiveGlobeRealModal.tsx', 'w') as f:
    f.write(text2
# Remove duplicates text !")

File: fix_types.mjs
import fs from 'fs';

let t = fs.readFileSync('src/components/ExhibitionModal.tsx', 'utf8');
t = t.replace(/(\s*"sculptures": "Sculpture",\n)+/g, '\n    "sculptures": "Sculpture",\n');
t = t.replace(/(\s*"photography": "Photography",\n)+/g, '\n    "photography": "Photography",\n');
t = t.replace(/(\s*"posters": "Posters",\n)+/g, '\n    "posters": "Posters",\n');
fs.writeFileSync('src/components/ExhibitionModal.tsx', t);

let t2 = fs.readFileSync('src/components/InteractiveGlobeMap/InteractiveGlobeRealModal.tsx', 'utf8');
t2 = t2.replace(/(\s*'egyptian-museum-cairo-collection': '\/data\/egyptian-museum-cairo-collection\.json',\n)+/g, "\n          'egyptian-museum-cairo-collection': '/data/egyptian-museum-cairo-collection.json',\n");
t2 = t2.replace(/(\s*'conde-paintings': '\/data\/musee-conde-collection\.json',\n)+/g, "\n          'conde-paintings': '/data/musee-conde-collection.json',\n");
fs.writeFileSync('src/components/InteractiveGlobeMap/InteractiveGlobeRealModal.tsx', t2);

console.log("Fixed duplicates!");